handlers/main.yml

```diff
@@ -5,10 +5,10 @@
     state: reloaded
   when:
     - openhpc_slurm_service is not none
-    - openhpc_slurm_service_enabled | bool
+    - openhpc_slurm_service_started | bool
 
 - name: Restart Munge service
```

Comment:
NB: the `noconfig` branch (PR #62) makes munge start first - this is necessary for correct slurm start